### ISSUE TITLE
Output spammed with exceptions

### ISSUE THREAD

**Issue by rapus95** [qouteall/ImmersivePortalsMod#25]
I'm getting LOTS of these NullPointerExceptions
```
[21:15:37] [main/FATAL]: Error executing task on Client
java.lang.NullPointerException: null
	at net.minecraft.class_1937.<init>(class_1937.java:127) ~[intermediary-minecraft-1.14.4-client.jar:?]
	at net.minecraft.class_638.<init>(class_638.java:80) ~[intermediary-minecraft-1.14.4-client.jar:?]
	at com.qouteall.immersive_portals.ClientWorldLoader.createFakedClientWorld(ClientWorldLoader.java:161) ~[immersive-portals-0.18.jar:?]
	at com.qouteall.immersive_portals.ClientWorldLoader.getOrCreateFakedWorld(ClientWorldLoader.java:92) ~[immersive-portals-0.18.jar:?]
	at com.qouteall.immersive_portals.my_util.Helper.loadClientWorld(Helper.java:421) ~[immersive-portals-0.18.jar:?]
	at com.qouteall.immersive_portals.MyNetworkClient.lambda$processRedirectedPacket$3(MyNetworkClient.java:190) ~[immersive-portals-0.18.jar:?]
	at net.minecraft.class_1255.method_18859(class_1255.java:144) [intermediary-minecraft-1.14.4-client.jar:?]
	at net.minecraft.class_4093.method_18859(class_4093.java:23) [intermediary-minecraft-1.14.4-client.jar:?]
	at net.minecraft.class_1255.method_16075(class_1255.java:118) [intermediary-minecraft-1.14.4-client.jar:?]
	at net.minecraft.class_1255.method_5383(class_1255.java:103) [intermediary-minecraft-1.14.4-client.jar:?]
	at net.minecraft.class_310.method_1523(class_310.java:915) [intermediary-minecraft-1.14.4-client.jar:?]
	at net.minecraft.class_310.method_1514(class_310.java:410) [intermediary-minecraft-1.14.4-client.jar:?]
	at net.minecraft.client.main.Main.main(Main.java:155) [intermediary-minecraft-1.14.4-client.jar:?]
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:1.8.0_201]
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62) ~[?:1.8.0_201]
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:1.8.0_201]
	at java.lang.reflect.Method.invoke(Method.java:498) ~[?:1.8.0_201]
	at net.fabricmc.loader.game.MinecraftGameProvider.launch(MinecraftGameProvider.java:178) [fabric-loader-0.6.1+build.165.jar:?]
	at net.fabricmc.loader.launch.knot.Knot.init(Knot.java:131) [fabric-loader-0.6.1+build.165.jar:?]
	at net.fabricmc.loader.launch.knot.KnotClient.main(KnotClient.java:26) [fabric-loader-0.6.1+build.165.jar:?]
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:1.8.0_201]
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62) ~[?:1.8.0_201]
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:1.8.0_201]
	at java.lang.reflect.Method.invoke(Method.java:498) ~[?:1.8.0_201]
	at org.multimc.onesix.OneSixLauncher.launchWithMainClass(OneSixLauncher.java:196) [NewLaunch.jar:?]
	at org.multimc.onesix.OneSixLauncher.launch(OneSixLauncher.java:231) [NewLaunch.jar:?]
	at org.multimc.EntryPoint.listen(EntryPoint.java:143) [NewLaunch.jar:?]
	at org.multimc.EntryPoint.main(EntryPoint.java:34) [NewLaunch.jar:?]
```

**Comment by qouteall** [comment]
Do you install mods that add new dimensions?

**Comment by rapus95** [comment]
yes, IIRC, ArcaneMagic.

**Comment by rapus95** [comment]
Update: This error occurs every 5 seconds

**Comment by qouteall** [comment]
Maybe this is caused by manual day time sync.

**Comment by qouteall** [comment]
This may be fixed in 0.20. Try it out
